airbyte-webapp/.storybook/main.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
     "@storybook/preset-create-react-app",
```

Comment:
:information_source: I've changed this file to be a `.js` file again. It seems the typing conflict with ts-node somehow. I was not able to quickly resolve that (custom `tsconfig.json`, `.babelrc`, changing this files syntax didn't work). Given that this file anyway has no real typing in it at all, I reverted it back to a .js file.